dataline-integrations/singer/postgres/destination/requirements.txt

```diff
@@ -1,2 +1,2 @@
 psycopg2-binary == 2.8.5
-target-postgres == 1.1.3
+singer-target-postgres === 0.2.4
```

Comment:
this package is much more maintained https://github.com/datamill-co/target-postgres